docs/reference/elasticsearch/index-settings/_snippets/serverless-availability.md

```diff
@@ -0,0 +1,3 @@
+::::{admonition} Settings supported in Serverless
```

Comment:
we might want to run this language and the language at the top of the serverless settings list page past the serverless pms (or maybe someone from search like serena). marketing/positioning around limitations is very sensitive. Liam would probably know who to ask.

Reply:
Sounds great, I'll be PTO until 26th, would you take care of sharing this as soon as you feel is the right time?
Feel free to make your own proposal or changes for the banner itself.

My 2 cents are:
- The link to the `differences` page is added for the user to understand better the `why` not all settings are available (there's a parallel PR that is going to improve that very soon, already approved and waiting for latest feedback). So, if we include this link it has to be over a text that clarifies that. That's why I used `restrict the available settings` in the text of the link to that page.

- On the other hand the other link is to show the actual list of available settings. This one is easier to address :)